docs-chef-io/content/inspec/cloud_security.md

```diff
@@ -0,0 +1,212 @@
+
+For inherited controls, you can rely on AWS to provide the security of the cloud itself. For customer-specific security tasks, the apps and configurations specific to your unique circumstances are running on the AWS platform. For example, planning on managing the security of your instances, including user access controls (IAM), managing patches in your operating systems and which firewall ports are open/closed in your instances.
+
+### Auditing Cloud Environments with Chef
+
+Chef scans systems across all environments (Dev, Pre-Production, Production), all systems (Cloud, Kubernetes, VMs, Containers, Windows, Linux), and all clouds (AWS, Azure, Google, Alibaba).
+
+You have access to the Chef InSpec resources library, including benchmarks and profiles created by the Center for Internet Security (CIS). The CIS benchmark turns security best practices into specific, actionable controls that can run against your cloud systems. In addition, the Chef CIS benchmarks prevent the misconfiguration in the first place itself. Thus, fixing the misconfigurations in the Infrastructure As Code (IaC) used to create the resources.
+
+The CIS/DISA STIG certified profiles are available for Amazon Web Services, Microsoft Azure, GCP, Alibaba Cloud, and Google Cloud/Compute platforms. These profiles provide insights about services like identity management, security groups, and storage buckets that are implemented securely. With Chef InSpec's easy-to-code language and built-in profiles and resources, you can address your organization's concerns. And with Chef Infra, you can take corrective actions based on the insights derived.
+
+Policy as code with provisioning tools like Terraform merges infrastructure-as-code and compliance-as-code into a single workflow. The Chef InSpec can be used with *Terraform* in two different ways to confirm compliance:
+
+- Audit provisioned infrastructure: When developing Terraform code for repositories, Chef InSpec can verify the resources are updated to match the tested and approved criteria.
+
+- Terraform code declaration: Chef InSpec declares the infrastructure configuration in the test-driven development, and Terraform can be used to provision resources accordingly. In this way, Terraform manages provisioning while InSpec ensures the provisioned resources meet the organization’s policy requirements.
+
+Together, these capabilities ensure that with Chef, you have a single way to detect, correct, and automate any issue in any cloud-based environment.
+
+### Docker Benchmark Recommendations
+
+The following Docker benchmark recommendations apply to both the host and the Docker components:
+
+- Host configuration: Secure the host on which the Docker engine runs, and the container it hosts is safe.
```

Comment:
That doesn't look right:

"Secure the host on which the Docker engine runs, and the container it hosts is safe."

Reply:
Sharan reviewed it. Will check again.